### PR TITLE
Update browser launch regex to support non-default logging frameworks

### PR DESCRIPTION
**Pull request by steveoh**
this regex only works with the default logging system. If you change the output template with something like serilog or another logging framework the regex will not match. Loosening it up to match this text will work in more cases.

### CODE REVIEW

**Comment by gregg-miskelly** [comment]
@steveoh You are just changing the documentation. Did you mean to change the default regex that the templates output also?

File: debugger-launchjson.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ The default launch.json template (as of C# extension v1.20.0) for ASP.NET Core p
 ```json
     "serverReadyAction": {
         "action": "openExternally",
-        "pattern": "^\\s*Now listening on:\\s+(https?://\\S+)"
+        "pattern": "\\s*Now listening on:\\s+(https?://\\S+)"
```
Comment (gregg-miskelly) [inline debugger-launchjson.md:48]: Assuming JavaScript supports it, this should probably be `\\bNow...` to make sure there is a word boundary. Can you test if that works in your scenario?

Comment (steveoh) [reply debugger-launchjson.md:48]: Sure I'll test it today. I didn't know the default template output this. I still had the launch browser property. I can update the template if you point me towards it.

**Comment by steveoh** [comment]
@gregg-miskelly i cant find the template file. So this is still only docs. Link me if you would like me to update it there also.

**Comment by gregg-miskelly** [comment]
Here are the other spots:
```
C:\dd\omnisharp-vscode\package.json(2016):                "pattern": "^\"^\\\\\\\\s*Now listening on:\\\\\\\\s+(https?://\\\\\\\\S+)\""
C:\dd\omnisharp-vscode\src\assets.ts(303):        "pattern": "^\\\\s*Now listening on:\\\\s+(https?://\\\\S+)"
```

**Comment by dnfgituser** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/OmniSharp/omnisharp-vscode?pullRequest=3842) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/OmniSharp/omnisharp-vscode?pullRequest=3842) before we can accept your contribution.<br/><br/>:x: steveoh [sign now](https://cla.dotnetfoundation.org/OmniSharp/omnisharp-vscode?pullRequest=3842)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/OmniSharp/omnisharp-vscode?pullRequest=3842) it.</sub>

**Comment by steveoh** [comment]
ok @gregg-miskelly please review those crazy escape sequence and let me know if i fouled it up.

**Comment by gregg-miskelly** [comment]
I will give it a quick test, but I think you have that right.

**Comment by gregg-miskelly** [comment]
Looks like you want to change the package.json line to :`"\\\\bNow listening on:\\\\s+(https?://\\\\S+)"`.

**Comment by steveoh** [comment]
Ok, i made that change. Hopefully that's everything but if not I gave you access to edit my branch. Thanks!

**Comment by gregg-miskelly** [comment]
Thanks for fixing this!

**Pull request merged**